src/config.ts

```diff
@@ -1,7 +1,7 @@
 // deploy the contract and fill in the contract address
 export const config = {
-  substrateEndpoint: "ws://127.0.0.1:9944",
-  psp22NativeAddr: "",
-  erc20NativeAddr: "",
-  erc20EvmAddr: "",
+  substrateEndpoint: 'ws://127.0.0.1:9944',
+  psp22NativeAddr: process.env.REACT_APP_PSP22_NATIVE_ADDRESS || '',
```

Comment:
I don't see the benefit for using a `.env` in this situation (I need to open a file and edit it anyway). So I'll remove this to keep things simple

Reply:
Sure, I just thought then we have to delete the addresses in the config file every time we push the code.